.appveyor.yml

```diff
@@ -1,35 +1,21 @@
 environment:
   matrix:
```

Comment:
Open question: shouldn't the node.js maintenance LTS (v4) be included in the matrix until [april 2018](https://github.com/nodejs/Release)?

Reply:
(this should not be an issue with [Buffer.from](https://nodejs.org/docs/latest-v4.x/api/buffer.html#buffer_class_method_buffer_from_str_encoding))

Reply:
As you want but I prefer not supporting those very old Node.js versions and encourage people to upgrade (Node 4 = IE6 of server side)

Reply:
Sounds good 👌